mdl.gemspec

```diff
@@ -11,6 +11,7 @@ Gem::Specification.new do |spec|
   spec.description = 'Style checker/lint tool for markdown files'
   spec.homepage = 'http://github.com/markdownlint/markdownlint'
   spec.license = 'MIT'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```

Comment:
should this be a string, or a bool?

Reply:
I copied it verbatim [from here](https://docs.rubocop.org/rubocop/cops_gemspec.html#gemspecrequiremfa).

Reply:
Nice. Just checking, wasn't sure.